device_coupler/runner.py

```diff
@@ -84,13 +84,19 @@ def _process_event_queue(self):
                 event = self._event_queue.get(timeout=self._WORKER_TIMEOUT)
                 self._logger.info(event)
                 if event.event_type == DiscoveryEventType.DISCOVERY:
-                    self._daq_client.process_device_discovery(event.mac, event.vlan)
+                    port = self._get_device_port(event.vlan)
+                    self._daq_client.process_device_discovery(event.mac, event.vlan, port)
                 else:
                     self._daq_client.process_device_expiry(event.mac)
-            except Empty:
+            except Exception:
                 # Worker timeout. Do nothing
                 pass
 
+    def _get_device_port(self, vlan):
+        """Temporary mapping for VLAN to ports uses first VLAN of accepted test VLAN range."""
```

Comment:
Why is this temporary?  Just indicate what it does without getting into how long it might exist :-)

Reply:
Changed.